src/services/account.validation.service.ts

```diff
@@ -104,6 +104,7 @@ export function mapResponseType(
             case "OK":
                 return {
                     status: "success",
+                    imageUrl: baseResult.fileName.toLowerCase().endsWith(".xhtml") ? `/xbrl_validate/render/${baseResult.fileId}`  : undefined,
```

Comment:
I think this would be better reveresed because all non-single submissions are ZIP files but I'm not convinced all single submissions use an `xhtml` extension. I have also seen `ixbrl` extension used. 
Also, if we could make the url for rendering a constant I think that would be better.
You can add the constant to the constants package.
```suggestion
                    imageUrl: baseResult.fileName.toLowerCase().endsWith(".zip) ? undefined : `${Urls.RENDER}/${baseResult.fileId}`,
```

Reply:
Thank you. I have made change, please have a look and give me your thoughts.